homework-unit7/chat/main.css

```diff
@@ -0,0 +1,252 @@
+*{
```

Comment:
```suggestion
* {
```

Trước dấu `{` hãy thêm 1 dấu cách